IBAnimatable/FadeAnimator.swift

```diff
@@ -20,12 +20,9 @@ public class FadeAnimator: NSObject, AnimatedTransitioning {
     self.transitionDuration = transitionDuration
 
     switch direction {
-    case .in:
-      self.transitionAnimationType = .fade(direction: .in)
-      self.reverseAnimationType = .fade(direction: .out)
-    case .out:
-      self.transitionAnimationType = .fade(direction: .out)
-      self.reverseAnimationType = .fade(direction: .in)
+    case .in, .out:
+      self.transitionAnimationType = .fade(direction: direction)
+      self.reverseAnimationType = .fade(direction: direction.opposite)
     default:
```

Comment:
Since `opposite` for the `.cross` case just returns `.cross` I think it can be removed completely 👍